datafusion/proto/src/to_proto.rs

```diff
@@ -1187,6 +1187,7 @@ impl TryFrom<&ScalarValue> for protobuf::ScalarValue {
                     Value::LargeBinaryValue(s.to_owned())
                 })
             }
+            scalar::ScalarValue::FixedSizeBinary(_, _) => todo!(),
```

Comment:
Could you please return a "NotImplemented" error here rather than a `todo()` panic?

Reply:
Done. The error type in to_proto doesn't have the NotImplented variant as it isn't a DataFusionError, hopefully a string saying that it isn't implemented is sufficient?